go.mod

```diff
@@ -1,5 +1,8 @@
 module github.com/nats-io/jwt
 
-require github.com/nats-io/nkeys v0.1.4
+require (
+	github.com/nats-io/nkeys v0.1.3
+	github.com/stretchr/testify v1.5.1 // indirect
```

Comment:
Hopefully that does not bring this dep in the nats-server/go client. I know you are a big fan of it, but I like to minimize any dependency unless really required. So far, all testing in nats-server/streaming-server have not needed any feature that testify may bring that could not be done with std lib.

Reply:
+1

Reply:
fixed.